app/css/style.css

```diff
@@ -8498,49 +8498,111 @@ path.state:hover {
   right: 190px;
   top: 160px; }
 
+<<<<<<< HEAD
```

Comment:
this can also be fixed by rm'ing this from the repo. this shouldn't be in the repo anyways